pkgs/desktops/mate/mate-themes/default.nix

```
@@ -1,18 +1,18 @@
-{ stdenv, fetchurl, pkgconfig, intltool, mate, gtk2, gtk3,
+{ stdenv, fetchurl, pkgconfig, intltool, mate-icon-theme, gtk2, gtk3,
   gtk_engines, gtk-engine-murrine, gdk-pixbuf, librsvg }:
 
 stdenv.mkDerivation rec {
   pname = "mate-themes";
-  version = "3.22.20";
+  version = "3.22.21";
 
   src = fetchurl {
     url = "https://pub.mate-desktop.org/releases/themes/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
-    sha256 = "0c3dhf8p9nc2maky4g9xr04iil9wwbdkmhpzynlc6lfg4ksqq2bx";
+    sha256 = "051g2vq817g84yrqzf7hjcqr4xrghnw1rprjd6jf5mhhzmwcas6n";
   };
 
   nativeBuildInputs = [ pkgconfig intltool gtk3 ];
```

```suggestion
  nativeBuildInputs = [ pkgconfig gettext gtk3 ];
```

migrate from intltool to gettext https://github.com/mate-desktop/mate-themes/releases/tag/v3.22.21

---

Done.